Exercise_D.js

```diff
@@ -14,7 +14,13 @@
 Use a combination of conditionals and loops to create your answer for this question. */
 
 function multiplyNumber(number) {
-  // CODE HERE
+  if (number <= 0 || isNaN(number) == true) {
+    return "ERROR";
+  } else if (number >= 1000000) {
+    return number;
+  } else {
+    return number*100000;
+  }
 }
 

```

Comment:
Most of this answer is correct.
We wanted you to use a loop to get the value over 1000000 not just times by that number.
In your else block you need to develop a while loop so that we can push the number value over 1000000